common/conf/env/PROD.properties

```diff
@@ -23,6 +23,7 @@ ajax.cors.origin=http://www.theguardian.com,https://www.theguardian.com,https://
 
 amp.host=amp.theguardian.com
 amp.scheme=https://
+amp.cors.origin=https://www.theguardian.com,https://amp.theguardian.com, https://cdn.ampproject.org
```

Comment:
Yes they did, we are reliant on it for our fonts as well. Other publishers are reliant on it as well so they will inform us if anything changes.